modules/distr-tracing-install-otel-operator.adoc

```diff
@@ -28,17 +32,17 @@ Do not install Community versions of the Operators. Community Operators are not
 
 . Navigate to *Operators* -> *OperatorHub*.
 
-. Type *distributing tracing datacollection* into the filter to locate the {OTELName} Operator.
+. Type *distributing tracing data collection* into the filter to locate the {OTELName} Operator.
 
 . Click the *{OTELName} Operator* provided by Red Hat to display information about the Operator.
 
 . Click *Install*.
 
-. On the *Install Operator* page, select the *stable* Update Channel. This automatically updates your Operator as new versions are released.
+. On the *Install Operator* page, accept the default *stable* Update channel. This automatically updates your Operator as new versions are released.
```

Comment:
Since you're lowering "channel", shouldn't "Update" be lowered too?

Reply:
The capitalization matches the current GUI.